README.md

```diff
@@ -1,7 +1,7 @@
 ## [Datakit](http://github.com/docker/datakit)
 
 [![Build Status (OSX, Linux)](https://travis-ci.org/docker/datakit.svg)](https://travis-ci.org/docker/datakit)
-[![Build status (Windows)](https://ci.appveyor.com/api/projects/status/x36ff2iv16xihftb/branch/master?svg=true)](https://ci.appveyor.com/project/samoht/datakit/branch/master)
+[![Build status (Windows)](https://ci.appveyor.com/api/projects/status/x36ff2iv16xihftb/branch/master?svg=true)](https://ci.appveyor.com/project/docker/datakit/branch/master)
```

Comment:
Should the SVG url be updated as well? https://ci.appveyor.com/api/projects/status/x36ff2iv16xihftb/branch/master?svg=true gives me

```
{"message":"Project not found or access denied."}
```